plugins/out_es/es.c

```diff
@@ -862,6 +862,7 @@ static void cb_es_flush(struct flb_event_chunk *event_chunk,
     }
     else {
         /* The request was issued successfully, validate the 'error' field */
+        flb_plg_error(ctx->ins, "Headers: \n%s", c->resp.data);
```

Comment:
I think this was added for debugging, let's remove it for prod release?

Reply:
Great. Good catch. Updated.